programs/gamma/src/states/pool.rs

```diff
@@ -123,10 +123,16 @@ pub struct PoolState {
     /// latest dynamic fee rate
     pub latest_dynamic_fee_rate: u64,
 
+    // if zero then default of 10%(100000) is used
+    pub max_trade_fee_rate: u64,
+
+    // if zero then default of 300_000 is used
+    pub volatility_factor: u64,
+
     // This will store the partner information, like how much token0 and token1 they was invested from their platforms.
     pub partners: [PartnerInfo; 1],
```

Comment:
this was intentionally added above the array.
currently all data is zero so this should be fine.